SCClassLibrary/Common/Math/SimpleNumber.sc

```diff
@@ -709,8 +711,7 @@ SimpleNumber : Number {
 
 		// min value of precision is 0.001; this ensures that we stick to 3 decimal places in the
```

Comment:
this comment is in the wrong place now. should be above `precision = max(precision, 0.001);` right?